### PR TITLE
[Defensive] Sort Command

### PR DESCRIPTION
**Pull request by lyeyixian**

### CODE REVIEW

File: src/main/java/seedu/address/logic/command/sort/SortAccommodationCommand.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
     public static final String MESSAGE_USAGE =
             "sort accommodation: Sorts the list of accommodation in a travel plan by the keyword input by the user.\n"
-                    + "Parameters: KEYWORD (COST/DATE)\n";
+                    + "Parameters: KEYWORD (cost/date)\n";
```
Comment (jeannetoh99) [inline src/main/java/seedu/address/logic/command/sort/SortAccommodationCommand.java:20]: Can add example to this?

File: src/main/java/seedu/address/logic/parser/SortCommandParser.java [hunk 2]
```diff
@@ -20,31 +22,36 @@
...
-            String sortKeyword = keywords[2];
+            String travelPlanObjectType = keywords[OBJECT_TYPE_POSITION].substring(1);
 
             switch (travelPlanObjectType) {
             case Activity.TPO_WORD:
-                return new SortActivityCommand(sortKeyword);
+                if (keywords.length < 3) {
+                    throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                            SortActivityCommand.MESSAGE_USAGE));
+                }
```
Comment (jeannetoh99) [inline src/main/java/seedu/address/logic/parser/SortCommandParser.java:34]: Should you check this outside of the switch case and just throw SortCommand.MESSAGE_USAGE instead?

**Review by jeannetoh99 — APPROVED** [review]

**Review by jeannetoh99 — APPROVED** [review]
Added some comments, otherwise LGTM.

File: src/main/java/seedu/address/logic/parser/SortCommandParser.java (hunk 2 above)
Comment (lyeyixian) [reply src/main/java/seedu/address/logic/parser/SortCommandParser.java:34]: I make it so its like 
"sort -activity " will show the error message specific to activity
"sort -accommodation" will show error message specific to accommodation etc.
do you think its ok?

**Review by jiaweiteo — APPROVED** [review]
LGTM

**Pull request merged**